apps/meteor/tests/end-to-end/api/24-methods.js

```diff
@@ -119,6 +121,307 @@ describe('Meteor.methods', function () {
 		});
 	});
 
+	describe('[@getReadReceipts]', () => {
```

Comment:
tests are looking awesome, congrats 🎉 

I'm just missing testing the behavior of `readThreads` method.. and also checking if it changes the "read state" of the room, as previously proposed in the other PR.